app/components/paperShow/refCitedPapers/desktopRefCitedPapers.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useLocation } from 'react-router-dom';
+import getQueryParamsObject from '../../../helpers/getQueryParamsObject';
+import RefCitedPaperList from './refCitedPaperList';
+import SearchContainer from './searchContainer';
+import { AppState } from '../../../reducers';
+import RefCitedPagination from './refCitedPagination';
+import { REF_CITED_CONTAINER_TYPE } from '../constants';
+import { useSelector } from 'react-redux';
+const useStyles = require('isomorphic-style-loader/useStyles');
+const styles = require('./referencePapers.scss');
+
+interface Props {
+  paperId: string;
+  type: REF_CITED_CONTAINER_TYPE;
+}
+
+const DesktopRefCitedPapers: React.FC<Props> = props => {
+  useStyles(styles);
+  const location = useLocation();
+  const paperIds = useSelector((state: AppState) => {
+    if (props.type === 'reference') {
+      return state.paperShow.referencePaperIds;
+    }
+    return state.paperShow.citedPaperIds;
+  });
+
+  const isLoading = useSelector((state: AppState) =>
+    props.type === 'reference' ? state.paperShow.isLoadingReferencePapers : state.paperShow.isLoadingCitedPapers
+  );
+
+  React.useEffect(() => {
+    if (isLoading && location.state?.scrollTo === props.type && wrapperNode.current) {
+      window.scrollTo(0, wrapperNode.current.offsetTop - 175);
```

Comment:
It will be easier to understand if you use comments to describe `175` or define it as a global variable.

Reply:
👍